src/SprykerEco/Zed/Heidelpay/Business/HeidelpayBusinessFactory.php

```diff
@@ -98,6 +114,30 @@ public function createAuthorizeTransactionHandler(): AuthorizeTransactionHandler
         );
     }
 
+    /**
+     * @return \SprykerEco\Zed\Heidelpay\Business\Payment\Transaction\Handler\AuthorizeOnRegistrationTransactionHandlerInterface
+     */
+    public function createAuthorizeOnRegistrationTransactionHandler()
```

Comment:
It would be great if we can add typehints in this file. It should not take long time